README.md

```diff
@@ -78,3 +78,17 @@ result.mm   # returns the Mimi MarginalModel
 
 marginal_temp = result.mm[:climatedynamics, :TATM]  # marginal results from the marginal model can be accessed like this
 ```
+
+### Pulse Size Details
+
+By default, MimiDICE2010 will calculate the SCC using a marginal emissions pulse of 10 GtCO2 spread over ten years, or 1 GtCO2 per year for ten years.  The SCC will always be returned in $ per ton CO2 since is normalized by this pulse size. This choice of pulse size and duration is a decision made based on experiments with stability of results and moving from continuous to discretized equations, and can be found described further in the literature around DICE.
```

Comment:
I think that is incorrect. The unit of `pulse_size` is GtC, not GtCO2, and the default parameter value is `1e10`, so the default pulse size is 10 GtC. It might make sense, though, to change the unit of `pulse_size` to GtCO2, just to have everything uniform?